src/clientsettingsdlg.cpp

```diff
@@ -164,13 +167,13 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     cbxROutChan->setAccessibleName ( tr ( "Right output channel selection combo box" ) );
 
     // enable OPUS64
-    chbEnableOPUS64->setWhatsThis ( "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
-                                    tr ( "If enabled, the support for very small network audio packets is activated. Very small "
-                                         "network packets are only actually used if the sound card buffer delay is smaller than " ) +
-                                    QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) +
-                                    tr ( " samples. The "
-                                         "smaller the network buffers, the lower the audio latency. But at the same time "
-                                         "the network load increases and the probability of audio dropouts also increases." ) );
+    chbEnableOPUS64->setWhatsThis (
+        "<b>" + tr ( "Enable Small Network Buffers" ) + ":</b> " +
+        QString ( tr ( "If enabled, the support for very small network audio packets is activated. Very small "
```

Comment:
```suggestion
        QString ( tr ( "Enables support for very small network audio packets. These "
```
BTW I recently expanded this wording based on some conversations on the forums. See  https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Software-Manual.md#enable-small-network-buffers